src/client/post/EditPost/EditPost.js

```diff
@@ -137,6 +137,20 @@ class EditPost extends Component {
     }
   }
 
+  componentDidUpdate(prevProps) {
+    const currDraft = this.props.draftPosts.find(d => d.draftId === this.props.draftId);
+    if (
+      get(this.props, 'draftId') !== get(prevProps, 'draftId') &&
+      currDraft &&
+      currDraft.jsonMetadata &&
+      currDraft.jsonMetadata.campaignId
```

Comment:
rewrite condition as:
`this.props.draftId !== prevProps.draftId && !!get(currDraft, ['jsonMetadata', 'campaignId'])`